backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5771480_sys_gh25448_AD_Table_Update.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE ad_table
+SET ischangelog='Y'
+WHERE ad_table_id IN (SELECT cc.ad_table_id FROM fix.AD_Table_Changelog_Config_26092025 cc)
+;
```

Comment:
```suggestion
;

UPDATE AD_SysConfig SET Value='N', Updatedby=100, Updated='2025-09-26 20:09'  WHERE Name='SYSTEM_INSERT_CHANGELOG' and Value!='N';
```

taking the hint from max, let's just make sure we don't write into the changelog on inserts